saltstack/hortonworks/salt/java/init.sls

```diff
@@ -34,12 +34,14 @@ install_openjdk:
   pkg.installed:
     - pkgs: {{ pillar['openjdk_packages'] }}
 
-{% if grains['os'] == 'RedHat' and grains['osmajorrelease'] | int == 8 %}
+{% if salt['environ.get']('OS') == 'redhat8' %}
 {% if salt['environ.get']('IMAGE_BURNING_TYPE') == 'base' or salt['environ.get']('STACK_VERSION').split('.') | map('int') | list >= '7.2.18'.split('.') | map('int') | list %}
 download_rhel8_repo:
   file.managed:
     - name: /etc/yum.repos.d/rhel8_cldr_mirrors.repo
-    - source: https://mirror.infra.cloudera.com/repos/rhel/server/8/{{ salt['environ.get']('RHEL_VERSION') }}/rhel{{ salt['environ.get']('RHEL_VERSION') }}_cldr_mirrors.repo
+    # This actually points to 8.10, not 8.8, however there's no JDK 21 in the 8.8 repo, so we're installing it
```

Comment:
Actually it points to the currently latest 8.x version, but great idea to add an explanation comment

Reply:
According to RedHat 8.10 is the final RHEL 8 version - there won't be a 8.11 ever, so that comment is correct IMO.

![image](https://github.com/user-attachments/assets/2b183164-cd4a-4983-a28c-28b90934e498)